caliban/cloud/core.py

```diff
@@ -18,31 +18,53 @@
 from __future__ import absolute_import, division, print_function
 
 import datetime
+from copy import deepcopy
 from pprint import pformat
+from subprocess import CalledProcessError, check_output
 from typing import Any, Dict, Iterable, List, Optional, Tuple
-from copy import deepcopy
 
 import tqdm
 from absl import logging
 from blessings import Terminal
 from google.oauth2 import service_account
+from google.oauth2.credentials import Credentials
 from googleapiclient import discovery
 from googleapiclient.errors import HttpError
-from sqlalchemy.orm import Session
 
 import caliban.cloud.types as ct
 import caliban.config as conf
 import caliban.docker as d
-import caliban.util as u
-
-from caliban.history.utils import (get_sql_engine, get_mem_engine,
-                                   session_scope, generate_container_spec,
-                                   create_experiments)
 import caliban.history.types as ht
+import caliban.util as u
+from caliban.history.utils import (create_experiments, generate_container_spec,
+                                   get_mem_engine, get_sql_engine,
+                                   session_scope)
 
 t = Terminal()
 
 
+def auth_access_token() -> Optional[str]:
```

Comment:
wonder if there is a way to get this without subprocess call?

Reply:
man, I tried for a while to find a way to do it without the subprocess call... I can't find anything! the only way is to read the sqlite database they keep. So, this is our method for now.